setup.py

```diff
@@ -32,16 +32,24 @@
 
 include_dirs = [numpy.get_include()]
 library_dirs = []
-libraries = ['gd']
+libraries = ['libgd']
```

Comment:
it's just "gd" on *nix -- the system adds the "lib" for you ...

Reply:
On my Windows setup, it did not work as is. Hence, I needed to add `lib`. I agree that I didn't investigate thoroughly. When I'll move to Linux I'll revisit.